dbconstraint/src/main/java/org/evomaster/dbconstraint/parser/jsql/JSqlVisitor.java

```diff
@@ -321,9 +321,22 @@ public void visit(NotEqualsTo notEqualsTo) {
         throw new RuntimeException("Extraction of condition not yet implemented");
     }
 
+    private static String QUOTE_CHAR = "\"";
+
+    private static boolean hasSurroundingQuotes(String str) {
+        return str.length()>1 && str.startsWith(QUOTE_CHAR) && str.endsWith(QUOTE_CHAR);
+    }
+
+    private static String removeSurroundingQuotes(String str) {
+        return str.substring(1,str.length()-1);
+    }
+
     @Override
     public void visit(Column column) {
         String columnName = column.getColumnName();
+        if (hasSurroundingQuotes(columnName)) {
+            columnName = removeSurroundingQuotes(columnName);
+        }
```

Comment:
let's discuss this

Reply:
Quotes are valid in SQL 
https://stackoverflow.com/questions/2901453/sql-standard-to-escape-column-names
We can conclude that it is something we might get when parsing column names.

Reply:
hmmm, looks like thgere was a misunderstanding... these comments should had been added in the code, not just here on GitHub :)

Reply:
Indeed. I will add it to the code explaining the scenario.